release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2511,3 +2511,67 @@ link:https://access.redhat.com/solutions/5756401[{product-title} 4.6.15 containe
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/5780541[{product-title} 4.6.16 container image list]
+
+[id="ocp-4-6-16-features"]
+==== Features
+
+[id="ocp-4-6-16-insights-operator-enhancements"]
+===== Enhancements to Insight Operator
+
+The Insights Operator now collects the following additional information:
+
+* Config files for `MachineConfigPool` objects
+* The number of install plans
+* Config files for generic control planes
+* The set of running Red Hat images on a cluster
+* Information about crashlooping pods in the `openshift*` namespaces
+
+This information is useful for troubleshooting. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1887759[*BZ#1887759*], link:https://bugzilla.redhat.com/show_bug.cgi?id=1889676[*BZ#1889676*], link:https://bugzilla.redhat.com/show_bug.cgi?id=1905031[*BZ#1905031*], and link:https://bugzilla.redhat.com/show_bug.cgi?id=1913645[*BZ#1913645*].
+
+[id="ocp-4-6-16-auto-vm-network-import"]
+===== Automatic selection of network type during VM migration
+
+When running the migration wizard from VMware to CNV, the bridged network type is now selected automatically for the interface bound to the pod network. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1887813[*BZ#1887813*].
+
+[id="ocp-4-6-16-bug-fixes"]
+==== Bug fixes
+
+* Previously, version information metadata was not produced during the build process on Windows binaries. This update generates Windows version information during the build process by Red Hat's `golang` compiler, and thus presents version information on Windows binaries. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1891892[*BZ#1891892*])
+
+* The Ironic API service that is embedded during installer-provisioned installation on bare metal now uses four worker nodes instead of eight. This change reduces the RAM usage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1899107[*BZ#1899107*])
+
+* Previously, changing the `serviceAccountIssuer` field of the authentication resource updated the `kubi-apiserver` to validate tokens with the new issuer and reject tokens with the previous issuer. Because `kubi-apiserver` does not support multiple issuers, changing the `serviceAccountIssuer` has the potential to disrupt applications relying on bound tokens. Unless an application is coded to request a new token when the existing token receives `401` responses from the `kubi-apiserver`, the use of an invalid token will continue until the hardware is restarted or the invalid token exceeds 80% of its duration (at which point the kubelet will request a new token).
```

Comment:
s/duration (at which point the kubelet will request a new token)./duration, at which point the kubelet will request a new token.